lib/src/constants/constant_text.dart

```diff
@@ -55,3 +55,19 @@ class ForgotPasswordText {
   static const String sendButton = "Send";
   static const String tabletLabel = "Start from Scratch";
 }
+
+class RecipeFeedText {
+  static const likes = 'Likes';
+  static const comments = 'Comments';
+  static const save = 'Save';
+  static const textButton = 'Create Recipe';
+  static const ofYourFollowersAreOnline = 'of your followers are online';
+  static const loadingFail = 'Loading Faild';
+  static const yearAgo = 'year ago';
+  static const monthAgo = 'month ago';
+  static const weekAgo = 'week ago';
+  static const dayAgo = 'day ago';
+  static const hourAgo = 'hour ago';
+  static const minuteAgo = 'minute ago';
+  static const secondAgo = 'second ago';
+}
```

Comment:
the error text should have more detail, no internet or no feed available for example, and I see the UI show "d ago, h ago, ..." instead of "day ago, hour ago,..."

Reply:
I don't know how to make the difference for minutes and months, because they are both written as 'm' :<

Reply:
In my case, I user 'm ago' for minute and ' month ago, months ago' for month, you can refer it or do something better